### PR TITLE
Support netstandard2.0 by not using ArrayPool for that tfw

### LINKED ISSUE #14: would netstandard2.0 support be possible for this library?
Hi there, awesome library! I am using it in various projects.

Would it be possible to support netstandard2.0 as well? I would love to use the html DSL for generation of the file scaffold for rendering plots in [Plotly.NET](https://github.com/plotly/Plotly.NET), it would make that part of the library way easier to customize and adapt. We plan to move to .net5 eventually, but so far we are stuck on netstandard2.0 and it would be awesome if you would consider supporting it if there are no major blockers. Cheers!

### PR DESCRIPTION
**Pull request by kMutagene**
Thanks to @TheAngryByrd. this should close #14

### CODE REVIEW

**Review by dustinmoris — DISMISSED** [review]
Awesome, many thanks!

**Review by dustinmoris — DISMISSED** [review]

**Comment by dustinmoris** [comment]
I'll let @TheAngryByrd also review this PR as it was his suggested solution and I would want a second pair of eyes before merging but otherwise looks great and thank you for it!

File: src/Giraffe.ViewEngine/Engine.fs
Comment (TheAngryByrd) [inline src/Giraffe.ViewEngine/Engine.fs]: Only Suggestion I have is if we wanted to remove the ifdef around the open statement, we could fully qualify here and in the next call.  That's my only suggestion, feel free to ignore it.

**Review by TheAngryByrd — DISMISSED** [review]
LGTM, super tiny bikeshed that's completely ignorable so feel free to merge.

File: src/Giraffe.ViewEngine/Engine.fs
Comment (kMutagene) [reply src/Giraffe.ViewEngine/Engine.fs]: Done ;)

**Review by TheAngryByrd — APPROVED** [review]
LGTM

**Comment by TheAngryByrd** [comment]
@dustinmoris this requires a second approval so I can't merge

**Review by dustinmoris — APPROVED** [review]

**Pull request merged**
